vertx-grpc-server/src/main/java/module-info.java

```diff
@@ -6,6 +6,7 @@
   requires io.vertx.codegen.api;
   requires io.netty.codec;
   requires io.netty.buffer;
+  requires com.google.protobuf;
```

Comment:
that is actually fine since vertx-grpc-common already requires it we already need this dependency